scripts/cnv_wdl/somatic/README.md

```diff
@@ -48,6 +48,7 @@ The reference used must be the same between PoN and case samples.
 - ``CNVSomaticPairWorkflow.targets`` -- (optional) Target file (NOT in bed format) that was used to describe the baits in capture (exome) samples.  Please run ``ConvertBedToTargetFile`` to convert a BED file to a target file.  If provided, then WES workflow will be run; otherwise, WGS workflow will be run.
 - ``CNVSomaticPairWorkflow.tumor_bam_idx`` -- File path or storage location (depending on backend) of the tumor BAM file index.
 - ``CNVSomaticPairWorkflow.tumor_bam`` -- File path or storage location (depending on backend) of the tumor BAM index.
+- ``CNVSomaticPairWorkflow.is_run_oncotator`` -- (optional default: false) Whether oncotator should be run on the called segment file.
```

Comment:
If this will only be run on CNV output, so perhaps it would be better to just put the task in the copy-ratio BAM workflow instead, rather than have a whole separate workflow for this single task?  If you only want to run it on the tumor, rather than both normal and tumor, I think that shouldn't be too hard to arrange.

Also, we currently don't recall the CNV output of ConvertACNVResults.  If you want to run Oncotator on that instead, you'd need to add the converted output as an output of the allele-fraction workflow and add another call to CallSegments.  However, since ACNV is optional, this will complicate the control flow a bit.  Let me know if you want to go this route.